### ISSUE TITLE
v3.4.2 - update datimefield value in "collection types" not working > generate Javascript ValidationError (d.toISOString is not a function)

### ISSUE THREAD

**Issue by Arlkpi** [strapi/strapi#9124]
### Describe the bug

I just updated Strapi to v3.4.2, and while testing to add/update/remove elements in my collection types, I noticed that when you try to modify a datetime field value, you cannot "save" your change and you have the following error in the browser console:
- ValidationError main.c18b9e5c.chunk.js:1:7515340
- TypeError: "d.toISOString is not a function"

### Steps to reproduce the behavior

1. Go to your 'collection types', and select one containing a 'datetime' field
2. Click on the record to update
3. Click on the datetime input field to open the calendar and select a date (you can even set the initial value).
4. Click on the "save" button
5. See error in the console log
6. Unfortunately the value is not saved!!

### Expected behavior

You are supposed to be able to modify datetime fields value without getting any error/warning in the console and make it works!

### Screenshots

![STP_v3 4 2_datetime_error](https://user-images.githubusercontent.com/31731877/104554002-dade6800-563b-11eb-9392-c55b701b563f.png)

Here under the model of my collection in `/api/olodocument/models/Olodocument.settings.json` (the same as the beginning with strapi_beta18):
```json
{
  "connection": "default",
  "collectionName": "olodocuments",
  "info": {
    "name": "olodocument",
    "description": ""
  },
  "options": {
    "increments": true,
    "timestamps": true,
    "comment": ""
  },
  "attributes": {
    "creation": {
      "required": true,
      "type": "datetime"
    },
    "nom": {
      "required": true,
      "type": "string"
    },
    "document": {
      "model": "file",
      "via": "related",
      "plugin": "upload",
      "required": true
    },
    "description": {
      "type": "string"
    },
    "categorie": {
      "enum": [
        "Juridique",
        "Contractuel",
        "Communiqué de presse",
        "Dossier de presse",
        "Plaquette",
        "Flyer",
        "Guide utilisateur",
        "Présentation financière",
        "Rapport annuel",
        "Offres",
        "Bilan RSE"
      ],
      "type": "enumeration"
    }
  }
}
```

### System

- Node.js version: v14.11.0
- NPM version:
- Strapi version: v3.4.2
- Database: MySQL
- Operating system: CentOs (but reproduced on Fedora too)

**Comment by Arlkpi** [comment]
FYI, this bug is not reproduced in Strapi v3.4.1

**Comment by Arlkpi** [comment]
Corrected in v3.4.3, thx for your reactivity!

**Comment by petersg83** [comment]
Thank you :) 
Closing the issue as it has been fixed in v3.4.3
